packages/caraml-auth-google/version.py

```diff
@@ -0,0 +1,2 @@
+# Do not update this field manually; this value gets updated automatically during build time
+VERSION = "0.0.0"
```

Comment:
We don't have any CI on this yet - shall we set it to `0.0.1` and come back to update this when we add the publish?

Reply:
Alright, sounds good!